src/qibolab/transpilers/placer.py

```diff
@@ -0,0 +1,260 @@
+import random
+
+import networkx as nx
+from qibo.config import log, raise_error
+from qibo.models import Circuit
+
+from qibolab.transpilers.abstract import Placer, create_circuit_repr
+
+
+def assert_placement(circuit: Circuit, layout: dict, verbose=False) -> bool:
+    """Checks if layout is correct and matches the number of qubits of the circuit.
+
+    Args:
+        circuit (qibo.models.Circuit): Circuit model to check.
```

Comment:
This documentation style for arguments is verbose, and it is hard to get all references properly placed (even because there is no tool checking them, other than Sphinx).

I would propose to use a more free-form style, w/o sections for arguments and returned values, describing them in words. You should be familiar also with this, because it is exactly what is done in Python standard library modules.

Regarding types: move them all as type hints. They will be checked to be correct and doubly useful.

Reply:
Fine for me. At this point, we should do a big scan of the documentation in order to standardize it. As far as I remember I saw this notation (the one I suggested) in many examples.

Reply:
So i keep as it is now?